t/40mtls.t

```diff
@@ -36,11 +43,11 @@ sub run_tests {
 
 sub run_tls_client {
     my $opts = shift;
-    my $resp = `curl $opts --cacert misc/test-ca/root/ca.crt https://127.0.0.1.xip.io:$tls_port`;
+    my $resp = `curl $opts --cacert misc/test-ca/root/ca.crt --silent --show-error https://127.0.0.1.xip.io:$tls_port`;
     return $resp;
 }
 
-sub spawn_server {
+sub spawn_h2o_server {
```

Comment:
This rename is just because `spawn_server` is exported by Util.pm and thus a warning "Subroutine spawn_server redefined"  is issued.